ansible/configs/just-a-bunch-of-nodes/default_vars_ec2.yml

```diff
@@ -23,3 +23,25 @@ aws_region: ap-southeast-2
 # exist in your aws account and the private key should exist on the local machine
 # you are provisioning from.
 #key_name: "default_key_name"
+
+bastion_instance_platform: Red Hat Enterprise Linux
+node_instance_platform: Red Hat Enterprise Linux
+#bastion_instance_platform: Linux/UNIX
+#node_instance_platform: Linux/UNIX
+
+# AWS on-demand capacity reservation
+agnosticd_aws_capacity_reservation:
```

Comment:
Should this be two variables? `agnosticd_aws_capacity_reservation_regions` and `agnosticd_aws_capacity_reservations`? Nested structures like this are often trouble in Ansible.

Reply:
agreed.

I pushed a change to split the variable